komga-webui/src/locales/en.json

```diff
@@ -519,33 +519,33 @@
     }
   },
   "error_codes": {
-    "ERR_1000": "File could not be accessed during analysis",
-    "ERR_1001": "Media type is not supported",
+    "ERR_1000": "Could access file during analysis",
+    "ERR_1001": "Unsupported media type",
     "ERR_1002": "Encrypted RAR archives are not supported",
     "ERR_1003": "Solid RAR archives are not supported",
     "ERR_1004": "Multi-Volume RAR archives are not supported",
-    "ERR_1005": "Unknown error while analyzing book",
```

Comment:
Too passive

Reply:
what does that mean ? I don't get this.

Reply:
I think I changed this back, so marking as resolved.